charts/web3signer-validators/values.yaml

```diff
@@ -71,16 +71,16 @@ image:
   pullPolicy: IfNotPresent
   prysm:
     repository: "gcr.io/prysmaticlabs/prysm/validator"
-    tag: "v2.1.0"
+    tag: "v2.1.4-rc.0"
   prysmGnosis:
     repository: "ghcr.io/gnosischain/gbc-prysm-validator"
-    tag: "v2.1.0-gbc"
+    tag: "v2.1.2-gbc"
   lighthouse:
     repository: "sigp/lighthouse"
-    tag: "v2.3.1"
+    tag: "v2.5.0"
   teku:
     repository: "consensys/teku"
-    tag: "22.5"
+    tag: "22.7.0"
 
 ## Credentials to fetch images from private registry
```

Comment:
Lets brush doc strings here. For example, following doesn't seem relevant anymore:

```
## If not specified, will connect to prysm/lighthouse beacon nodes
## deployed with this helm chart
```

```
## How many validators to run
## **NB! Every validator hosts up to 100 keys in Mainnet and 1000 keys in Gnosis,
## so the number of validators must be >= (total number of validator keys) / max number of keys per validator
##
```